runtime/compiler/control/OptionsPostRestore.cpp

```diff
@@ -261,6 +263,13 @@ J9::OptionsPostRestore::iterateOverExternalOptions()
             }
             break;
 
+         case J9::ExternalOptions::XShareclassesDisableOnRestore:
+            {
+            if (FIND_ARG_IN_RESTORE_ARGS(OPTIONAL_LIST_MATCH, optString, 0) >= 0)
```

Comment:
At the moment, running with `-Xshareclasses:disableOnRestore` will fail on restore (unless I change this to `FIND_AND_CONSUME_RESTORE_ARGS`). I'm expecting the VM to use the `FIND_AND_CONSUME` variant when they process this option.

fyi @hangshao0